app/celery/tasks.py

```diff
@@ -527,71 +522,6 @@ def check_billable_units(notification_update):
         )
 
 
-@notify_celery.task(bind=True, name="send-inbound-sms", max_retries=5, default_retry_delay=300)
-def send_inbound_sms_to_service(self, inbound_sms_id, service_id):
-    inbound_api = get_service_inbound_api_for_service(service_id=service_id)
-    if not inbound_api:
-        # No API data has been set for this service
-        return
-
-    inbound_sms = dao_get_inbound_sms_by_id(service_id=service_id, inbound_id=inbound_sms_id)
-    data = {
-        "id": str(inbound_sms.id),
-        # TODO: should we be validating and formatting the phone number here?
-        "source_number": inbound_sms.user_number,
-        "destination_number": inbound_sms.notify_number,
-        "message": inbound_sms.content,
-        "date_received": inbound_sms.provider_date.strftime(DATETIME_FORMAT),
-    }
-
-    try:
-        response = request(
-            method="POST",
-            url=inbound_api.url,
-            data=json.dumps(data),
-            headers={"Content-Type": "application/json", "Authorization": "Bearer {}".format(inbound_api.bearer_token)},
-            timeout=60,
-        )
-        current_app.logger.debug(
-            "send_inbound_sms_to_service sending %s to %s, response %s",
-            inbound_sms_id,
-            inbound_api.url,
-            response.status_code,
-        )
-        response.raise_for_status()
-    except RequestException as e:
-        current_app.logger.warning(
-            "send_inbound_sms_to_service failed for service_id: %s for inbound_sms_id: %s and url: %s. exception: %s",
```

Comment:
I find having the `service_id` in the log message like we had here helpful because it means that we can easily identify the services with failing callbacks from the logs. Without this, we'd have to go and look up the service in the database. Is there a way we can add this back if others also find it useful?

Reply:
Just pushed up a commit that should help with this